server/service/src/sync/test/pull_and_push.rs

```diff
@@ -84,20 +84,16 @@ async fn test_sync_pull_and_push() {
         .changelogs(push_cursor, 100000, None /*change_log_filter*/)
         .unwrap();
     // Translate
-    let mut translated = vec![
-        translate_changelogs_to_sync_records(
-            &connection,
-            changelogs.clone(),
+    let mut translated = vec![translate_changelogs_to_sync_records(
+        &connection,
+        changelogs.clone(),
+        vec![
             ToSyncRecordTranslationType::PushToLegacyCentral,
-        )
-        .unwrap(),
-        translate_changelogs_to_sync_records(
-            &connection,
-            changelogs.clone(),
             ToSyncRecordTranslationType::PullFromOmSupplyCentral,
```

Comment:
Ugh, reminder that we need to clean up this test, definitely shouldn't have `PullFrom` in the push step! #5866

Reply:
I haven't fully digested that issue, but it feel like it should be fixed with this change, since this test is meant to just check we serialising correctly

Reply:
I think this helps, but still shouldn't have to include `pull` records in `push` step... think just a setup issue?